src/controllers/pokestop.js

```diff
@@ -237,8 +237,8 @@ class Invasion extends Controller {
 								if (!data.genderDataEng) {
 									data.genderDataEng = { name: '', emoji: '' }
 								}
-								if (this.GameData.utilData.types[gruntType.type]) {
-									data.gruntTypeEmoji = translator.translate(this.emojiLookup.lookup(this.GameData.utilData.types[gruntType.type].emoji, platform))
+								if (this.GameData.utilData.types[gruntType.type === 'Metal' ? 'Steel' : gruntType.type]) {
```

Comment:
Looks like the type is also used for colour below,
maybe
`const t = gruntType.type === 'Metal' ? 'Steel' : gruntType.type` then use t in the following code?